src/components/structures/TimelinePanel.js

```diff
@@ -503,7 +503,9 @@ var TimelinePanel = React.createClass({
         // This happens on user_activity_end which is delayed, and it's
         // very possible have logged out within that timeframe, so check
         // we still have a client.
-        if (!MatrixClientPeg.get()) return;
+        const cli = MatrixClientPeg.get();
+        // if no client or client is guest don't send RR (vector-im/riot-web#3758)
```

Comment:
If the issue was more informative, I'd be tempted to reference it but in this case I think it's pretty self-explanatory.